examples/rtlogger_helpers.py

```diff
@@ -0,0 +1,106 @@
+from zhinst.toolkit import Session
+from zhinst.toolkit.nodetree import Node
+from zhinst.toolkit.driver.nodes.awg import AWG
+
+
+def reset_and_enable_rtlogger(rtlogger: Node):
```

Comment:
```suggestion
def reset_and_enable_rtlogger(rtlogger: Node) -> None:
```
 It is good practice to specify the return type, even if it's None. Otherwise, it could have simply been forgotten.